### DISCUSSION TITLE
unified inbox for snappymail possible?

### DISCUSSION THREAD

**Discussion by europacafe** [the-djmaze/snappymail#1693] [Q&A]
I'm using snappymail docker + stalwart mail server. I have multiple email accounts (all with the same domain.tld).

As snappymail does not support a unified mailbox, is there any workaround to let me see all incoming emails from all my email accounts in one inbox?

**Comment by the-djmaze** [comment]
Not in one mailbox, but with Dovecot shared mailboxes and ACL rules you could see more (like in the demo).
![afbeelding](https://github.com/user-attachments/assets/bafe0a82-8f42-4428-8a97-3e7edabd1b2e)
